calicolipidlibrary/MIP2C.py

```diff
@@ -45,40 +45,6 @@ def theoreticalDigest(self):
 			return(FRAGMENTS)
 
 
-		@classmethod
-		def generateLibrary(cls,target=None, mode="pos"):
-			if target: handle = open(target, 'a+')
-			if mode == "pos":  
-				adduct_set = self.pos_adduct_set
-			elif mode =="neg":
-				adduct_set = self.neg_adduct_set
-			for c1 in self.chain1_ranges:
-				for c2 in self.chain2_ranges:
-					for adduct in adduct_set:
-						x  = MIP2C("MIP2C",[c1[0],c2[0]], [c1[1],c2[1]], [c1[2],c2[2]],  adduct=adduct)						
-						content = x.printNist()
-						if target: handle.write(content)
-						else: sys.stdout.write(content)
-			if target: handle.close()
-
-
-
-		@classmethod
-		def generateLibrary(cls,target=None, mode="pos"):
-			if target: handle = open(target, 'a+')
-			if mode == "pos":  
-				adduct_set = cls.pos_adduct_set
-			elif mode =="neg":
-				adduct_set = cls.neg_adduct_set
-			parent = cls.__bases__[0]
-			class_name = cls.__name__ 
-			for c in parent.chain_sets:
-				for adduct in adduct_set:
-					x  = cls(class_name, c, adduct=adduct)						
```

Comment:
Note the second generateLibrary definition removed the customized `chains` variable defined in the first generateLibrary definition.